Justfile

```diff
@@ -18,5 +18,5 @@ schemas:
 
   # Copies test vectors from the tbdex submodule repo into the vectors dir.
 vectors:
-  @git submodule update --init --recursive
+  @git submodule update --remote --merge
```

Comment:
you may have told me this but what's the reason for doing this again?

Reply:
oops, forgot to revert before committing. I couldn't get this command to pull the latest vectors without this change!